### ISSUE TITLE
`posTagSet`, `dependencySet`, etc in "contains" map

### ISSUE THREAD

**Issue by keighrim** [lapps/org.lappsgrid.serialization#42]
Current version of the API (2.5.0) does not support putting values for those fields. 
https://github.com/lapps/org.lappsgrid.serialization/blob/v2.5.0/src/main/groovy/org/lappsgrid/serialization/lif/View.groovy#L154-L162

(POS::posTagSet, NE::namedEntityCategorySet, Dependency::dependencySet, Parser::categorySet)

**Comment by keighrim** [comment]
I don't thinks there's a trivial way to make these rather arbitrary keys available to only certain `name`s of contained types. But we might want to provide a better way than hacking into the groovy map directly.

**Comment by ksuderman** [comment]
Isn't this what you are looking for?

https://github.com/lapps/org.lappsgrid.serialization/blob/v2.5.0/src/main/groovy/org/lappsgrid/serialization/lif/View.groovy#L84-L90

Since we don't know what users (including us) may be putting in the metadata at some point we have to hack around with maps.

**Comment by keighrim** [comment]
I was thinking of using this https://github.com/lapps/org.lappsgrid.serialization/blob/v2.5.0/src/main/groovy/org/lappsgrid/serialization/lif/View.groovy#L144-L146 
as `contains` is a nested map inside the `metadata` of a view. 

Anyway, I think either would be enough at this point. I'll close the issue.

**Comment by ksuderman** [comment]
If you use `View.addContains()` you get the `Contains` object that you can add more fields to.  At least it might be a bit better:

```
Contains c = view.addContains(Uri.NE, "producer", "type")
c.put("namedEntityCategorySet", "stanford")
```

However, it might be a good idea to have helper methods for at least the properties we define in our own vocabulary.  I just didn't want to add any until we had a better sense of what we wanted.

**Comment by keighrim** [comment]
Helper methods are available via #45 #47. Closing the issue.
